src/presets/index.ts

```diff
@@ -1,3 +1,2 @@
 export { default as node } from "./node";
 export { default as nodeless } from "./nodeless";
-export { default as cloudflare } from "./cloudflare";
```

Comment:
Hey @pi0, do you think it would be possible to keep the cloudflare preset here for one more RC and remove it in the next one? Maybe moving the file to `node/$cloudflare-<module>.ts`.

This might allow for a smoother transition?